### ISSUE TITLE
Tutorial job is broken on Jenkins since 20 days

### ISSUE THREAD

**Issue by damienmg** [bazelbuild/bazel#582]
Interesting part of the build:

```
____[6 / 19] Compiling external/bazel_tools/third_party/ijar/zip_main.cc
ERROR: /private/var/tmp/_bazel_ci/d77cd962f042e8a4ee37cc08700e23c8/external/bazel_tools/third_party/ijar/BUILD:29:1: undeclared inclusion(s) in rule '@bazel_tools//third_party/ijar:zipper':
this rule is missing dependency declarations for the following files included by 'external/bazel_tools/third_party/ijar/zip_main.cc':
  '/Users/ci/workspace/Tutorial/PLATFORM_NAME/darwin-x86_64/bazel-install/base_workspace/third_party/ijar/zip.h'
  '/Users/ci/workspace/Tutorial/PLATFORM_NAME/darwin-x86_64/bazel-install/base_workspace/third_party/ijar/common.h'.
____Building complete.
Target //backend:backend failed to build
Use --verbose_failures to see the command lines of failed build steps.
```

Looks related to @bazel_tools & hdr_checks. 

http://ci.bazel.io/job/Tutorial/PLATFORM_NAME=darwin-x86_64/81/console

**Comment by kchodorow** [comment]
@ulfjack has a patch that fixes this.

**Comment by damienmg** [comment]
Fixed in latest build \o/

**Comment by kchodorow** [comment]
Fixed via #445.
